orttraining/pytorch_frontend_examples/nvidia-bert/ort_supplement/ort_supplement.py

```diff
@@ -0,0 +1,128 @@
+import onnx
```

Comment:
should we add copyright headers?

Reply:
Added copyrights to new PR https://github.com/microsoft/onnxruntime-training-examples/pull/2